### PR TITLE
feat/Implemented generate templates command

### PR DESCRIPTION
**Pull request by BAntonellini**

### CODE REVIEW

File: dbt_coves/tasks/generate/templates.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    def run(self):
+        coves_config_folder = self.coves_config.get_config_folder()
+        templates_dest_path = Path(coves_config_folder / "templates")
+        templates_dest_path.mkdir(parents=True, exist_ok=True)
+
+        for template in self.DBT_COVES_TEMPLATES:
+            template_path = templates_dest_path / template
+            template_req = requests.get(
+                f"https://raw.githubusercontent.com/datacoves/dbt-coves/main/dbt_coves/templates/{template}"
```
Comment (ssassi) [inline dbt_coves/tasks/generate/templates.py:45]: You already have the files installed

File: dbt_coves/tasks/generate/templates.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+from .base import BaseGenerateTask
+
+console = console.Console()
+
+
+class GenerateTemplatesException(Exception):
+    pass
+
+
+class GenerateTemplatesTask(BaseGenerateTask):
```
Comment (ssassi) [inline dbt_coves/tasks/generate/templates.py:18]: It looks like you're not using any feature from `BaseGenerateTask`. Why not just inherit from `BaseTask`?

File: dbt_coves/tasks/generate/templates.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+        templates_destination_path = Path(dbtcoves_config_folder / "templates")
+        templates_destination_path.mkdir(parents=True, exist_ok=True)
+
+        for dbtcoves_template in glob.glob(f"{dbtcoves_templates_path}/*.*"):
+            template_path = Path(dbtcoves_template)
+            template_name = template_path.name
+            template_destination = templates_destination_path / template_name
+            try:
+                shutil.copyfile(template_path, template_destination)
```
Comment (ssassi) [inline dbt_coves/tasks/generate/templates.py:45]: It should not override files, but also not fail if already exists.

Comment (BAntonellini) [reply dbt_coves/tasks/generate/templates.py:45]: @ssassi why not overriding files? If we change any template in `dbt_coves` shouldn't the user be able to update his/hers by running `generate templates` again?

File: dbt_coves/tasks/generate/templates.py (hunk 2 above)
Comment (BAntonellini) [reply dbt_coves/tasks/generate/templates.py:18]: @ssassi you are right, but I think it should be from `BaseConfiguredTask` as we decided (as I recall) the user must have `.dbt_coves` folder

Let me know if not, as we should create it if missing, using `cwd` or getting `dbt project dir/`

File: dbt_coves/tasks/generate/templates.py (hunk 3 above)
Comment (ssassi) [reply dbt_coves/tasks/generate/templates.py:45]: ... and loosing any possible change he/she could have made? We could ask for confirmation, ir just not override them, they just need to remove/rename the old version.

File: dbt_coves/tasks/generate/templates.py (hunk 2 above)
Comment (ssassi) [reply dbt_coves/tasks/generate/templates.py:18]: The `.dbt_coves` folder should be always optional.

**Pull request merged**